staging/publishing/import-restrictions.yaml

```diff
@@ -56,6 +56,8 @@
   allowedImports:
   - k8s.io/apimachinery
   - k8s.io/component-base
+  - k8s.io/client-go
+  - k8s.io/client-go/tools/
```

Comment:
is this necessary?

Reply:
Yes, as I understand it 'k8s.io/client-go/tools/' is an explicitly restricted directory.